### ISSUE TITLE
Define dataset description ontology

### ISSUE THREAD

**Issue by ddeboer** [CLARIAH/clariah-plus#66]
An example is https://netwerk-digitaal-erfgoed.github.io/requirements-datasets/#dataset-example. Note that this ontology will only be used by the CLARIAH harvester internally rather than prescribed to dataset description publishers (as in the case of NDE).

**Comment by jblom** [comment]
@ddeboer yes we should not mix-up this internal ontology (actually data model) with something we **publish** to the world, so we can keep it very to the point and  simple (and gradually extend it later on; also possible to be able to **publish** to more elaborate  standards).

**Comment by ddeboer** [comment]
@jblom Actually I intended this process:

1. Harvest dataset descriptions from heterogenous CLARIAH partner registries.
2. Map the heterogenous data to the CLARIAH data model and store that in the CLARIAH Dataset Registry.
3. Expose dataset descriptions in the CLARIAH data model to the outside world (including Ineo).
